cluster/charts/rook-ceph/templates/namespace.yaml

```diff
@@ -0,0 +1,13 @@
+{{- if .Values.namespaceDivisionEnable }}
```

Comment:
What do you think about assigning `clusterNamespace` an empty string by default, then this you could simply compare if it's `ne ""`, and not introduce a new boolean.

Reply:
Thank you for your comment!
That would be more simple, so I will fix this.

Reply:
As [we can use null](https://v2.helm.sh/docs/chart_template_guide/#if-else), How about using null instead of ""?
I think it looks obvious that the value is not set.

If we use null, then `namespace.yaml` would be rewrited as follows.
```diff
--- a/cluster/charts/rook-ceph/templates/namespace.yaml
+++ b/cluster/charts/rook-ceph/templates/namespace.yaml
@@ -1,4 +1,4 @@
-{{- if (ne .Values.clusterNamespace  "") }}
+{{- if .Values.clusterNamespace }}
```

Reply:
Sorry, that change above does not pass `helm lint --strict`.
So, `clusterNamespace` should use "".